Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            //main
+            Interview();
+        }
+
+        public static void Interview()
+        {
+            //  gets all the input from the user, if one of them is invalid
+            //  it displays the relevant error code and returns
+            string name = PromptString("Name: ");
+            if (ErrorString(name, "Name cannot be empty.")) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4876897)